frontend/src/metabase/core/components/FormField/FormField.styled.tsx

```diff
@@ -77,3 +62,24 @@ export const FieldInfoLabel = styled.div`
   margin-left: auto;
   cursor: default;
 `;
+
+export interface FieldRootProps {
+  orientation: FieldOrientation;
+}
+
+export const FieldRoot = styled.div<FieldRootProps>`
```

Comment:
There was a bug where description become red when the field is invalid. This change fixes this issue